test_schemas/en/test_checkbox.json

```diff
@@ -150,20 +149,45 @@
                                         }
                                     }
                                 ],
-                                "description": "",
                                 "id": "non-mandatory-checkbox-question",
                                 "title": "What extra toppings would you like?",
                                 "type": "General"
                             },
                             "routing_rules": []
                         },
+                        {
+                            "type": "Question",
+                            "id": "single-checkbox",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "single-checkbox-answer",
+                                        "mandatory": false,
+                                        "options": [
+                                            {
+                                                "label": "Estimate",
+                                                "value": "Estimate"
+                                            }
+                                        ],
+                                        "type": "Checkbox",
+                                        "validation": {
+                                            "messages": {}
+                                        }
+                                    }
+                                ],
+                                "id": "single-checkbox-question",
+                                "title": "Is this an estimate?",
+                                "type": "General"
+                            },
+                            "routing_rules": []
```

Comment:
I know we are not good at this in existing schema but we should added keys that we dont need.